server/controllers/index.js

```diff
@@ -0,0 +1,3 @@
+import PostController from './postcontroller';
+
+export default PostController;
```

Comment:
@Chiazokam are you saying we agreed to have different files for each controller method?

Reply:
Yeah....ain't that? Please let somebody else confirm

Reply:
Yeah, regarding this, i used export default just for eslint to keep quiet. I cant use the {postcontrolller} style since i'm exporting only one method. Also, the idea is to have everything regarding posts in one file. That is, create post, delete post, get all posts would be in the PostController file. Anyone adding an import to the index file can simply remove the 'default' and wrap everything in an object.

Reply:
@Chiazokam if we are using different files then the method he used is okay.